sbi-spec/src/binary.rs

```diff
@@ -794,6 +804,20 @@ pub struct HartMask {
     hart_mask_base: usize,
 }
 
+/// Iteration for HartMask, from low to high.
+pub struct HartMaskIter {
```

Comment:
Please consider derive `Clone`, `Copy`, `Debug`, `Eq`, `PartialEq` and `Hash` for this structure. We exclude `Ord` and `PartialOrd` for that this structure does not define any orderings.

Reply:
Derived, thanks for point it out.